vng_api_common/models.py

```diff
@@ -166,9 +84,9 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
     @classmethod
-    def get_client(cls) -> Optional[Client]:
+    def get_client(cls) -> APIClient | Any | None:
         """
         Construct a client, prepared with the required auth.
         """
         config = cls.get_solo()
-        return _get_client(config.api_root, url_is_api_root=True)
+        return _get_client(config.api_root)
```

Comment:
This now fails when checking the config for autorisaties API for instance, because `build_client` expects a `Service`, not an `api_root`:

```
File "/home/steven/Documents/commonground-api-common/vng_api_common/views.py", line 104, in get_context_data
    config += _test_ac_config()
              ^^^^^^^^^^^^^^^^^
  File "/home/steven/Documents/commonground-api-common/vng_api_common/views.py", line 132, in _test_ac_config
    ac_client = AuthorizationsConfig.get_client()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/steven/Documents/commonground-api-common/vng_api_common/models.py", line 92, in get_client
    return _get_client(config.api_root)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/steven/Documents/commonground-api-common/vng_api_common/client.py", line 89, in get_client
    return build_client(service.api_root, client_factory=client_class)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/steven/Documents/VNG/open-notificaties/env/lib/python3.11/site-packages/zgw_consumers/client.py", line 30, in build_client
    config_adapter, nlx_base_url=service.nlx, **kwargs
                                 ^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'nlx'
```